.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,34 +1,26 @@
-  적절한 사이즈로 첨부하는 코드 👇
-  <img width="300" alt="" src="이미지URL">  
--->
-
-<br>
-
-## Other information 🔥
-<!-- 다른 리뷰어가 참고하면 좋을 내용을 알려주세요. 기타 참고사항이 있다면 작성해줍니다. -->
+<!-- UI 작업일 경우 뷰 스크린샷 첨부 -->
+<!-- <img width="300" alt="스크린샷 설명" src="이미지URL"> -->
 
+### 🚨 Checklist
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)